src/components/Codeblock.tsx

```diff
@@ -213,6 +214,15 @@ const Codeblock: React.FC<IProps> = ({
   codeLanguage,
   fromHomepage = false,
 }) => {
+  useEffect(() => {
+    
+    ;(typeof global !== "undefined" ? global : window).Prism = Prism
+
+    // Default languages included:
+    // https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js
+    require("prismjs/components/prism-solidity")
```

Comment:
Any reason we can't use `import "prismjs/components/prism-solidity"` at the top?

Reply:
Never mind, causes `Prism` is not defined error